apps/android/app/src/main/java/chat/simplex/app/model/SimpleXAPI.kt

```diff
@@ -1279,7 +1279,7 @@ sealed class ChatPagination {
 
   companion object {
     const val INITIAL_COUNT = 100
-    const val PRELOAD_COUNT = 20
+    const val PRELOAD_COUNT = 50
     const val UNTIL_PRELOAD_COUNT = 10
```

Comment:
```suggestion
    const val PRELOAD_COUNT = 100
    const val UNTIL_PRELOAD_COUNT = 50
```

because it's bumping into the end on long chats

Reply:
... and the cost of query is not proportionally higher to the number of loaded items, the large part of the cost is a "set up cost" that depends on the chat size, so it's better to load more items, less frequently, and start loading earlier...